### PR TITLE
CLDR-6396 quell some tests due to basicness

### PR DESCRIPTION
**Pull request by srl295**
depends on #2509  - draft. 

CLDR-6396

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
@srl295 where does this stand? I'm not sure how to review since it includes changes already made in the previous pr...

**Comment by btangmu** [comment]
btw I have changes (calling isLocaleAtLeastBasic) locally for
tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAttributeValues.java
tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestBasic.java

**Comment by srl295** [comment]
i'll rebase

**Comment by srl295** [comment]
@btangmu  try now

**Comment by srl295** [comment]
> @srl295 where does this stand? I'm not sure how to review since it includes changes already made in the previous pr...

They were in separate commits

**Comment by srl295** [comment]
> btw I have changes (calling isLocaleAtLeastBasic) locally for
> tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAttributeValues.java
> tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestBasic.java

you're welcome to just merge this in then to your local changes if you want. it's a small set of changes

**Review by btangmu — APPROVED** [review]

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java [hunk 1]
```diff
@@ -97,7 +97,11 @@ public void TestLanguageNameCoverage() {
         LanguageTagParser ltp = new LanguageTagParser();
         for (String locale : factory.getAvailableLanguages()) {
             String language = ltp.set(locale).getLanguage();
-            if (language.equals("root")) language = "und";
+            if (language.equals("root")) {
+                language = "und";
+            } else if(!StandardCodes.isLocaleAtLeastBasic(language)) {
+                continue;
```
Comment (btangmu) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java:103]: this prevents one failure, `regularPlus.containsAll(mainLocales)`

problem is, it not only skips locales that were in seed (like "aa"), but also locales that were in common, like "agq" -- consequently it results in a new failure, `localesForNames.containsAll(coverageLocales)`

**Pull request merged**